index.html

```diff
@@ -135,7 +135,26 @@ <h4 class="skills">Skills</h4>
             <hr>
             
         </div>
-    </section> 
+    </section>    
+    <section class="contact">
+        <fieldset class="contact-form">
+            <div class="background-img">
+                <h2 class="contact-heading">Contact me</h2>
+                <p class="contact-details">If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
+                    I'd love to help with it!</p>
+            <form action="" method="Post" class="details-form">
```

Comment:
- [ ] Please make sure that your form uses the `action` and `method` attributes to integrate with [formspree service](https://formspree.io/html). The `action` value should be similar to `https://formspree.io/f/{form_id}` and have the `method` value equal `POST` :blush: